sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -151,9 +151,10 @@ case class RowDataSourceScanExec(
 
     Map(
       "ReadSchema" -> requiredSchema.catalogString,
-      "PushedFilters" -> seqToString(markedFilters.toSeq),
-      "PushedAggregates" -> aggString,
-      "PushedGroupby" -> groupByString) ++
+      "PushedFilters" -> seqToString(markedFilters.toSeq)) ++
+      pushedDownOperators.aggregation.fold(Map[String, String]()) { v =>
```

Comment:
hmm is this change necessary? are `aggString` and `groupByString` still used afterwards?

Reply:
@sunchao - forget to remove `aggString` and `groupByString`. Removed. I think it'd better to align with `PushedLimit` and `PushedSample`. It's a minor change so packed here. I can make another minor PR if needed.

Reply:
Got it. This looks fine to me.

Reply:
The change looks good. I plan to fix this but didn't have a chance to do it yet. One problem, though. In the tests I check if the aggregate pushes down by checking the keywords in explain. If the explain contains `PushedAggregation: []`, then the aggregate is not pushed down. Since now we don't have the `[]`, some of the tests will fail. We probably should change all the tests to use a different method (collect aggregates in optimizedPlan) to check if aggregate push down. Probably do this in a separate PR.

Reply:
IMO it may be better to keep `[]` for empty aggregate/groupby, to be consistent with `PushedFilters` and those in `OrcScan`/`ParquetScan`.

Reply:
@huaxingao - ah yes I forgot about the test. I am leaving `RowDataSourceScanExec` as it is for now, and we can discuss later, thanks for comment @sunchao and @huaxingao.